src/schema/v2/artwork/artworkConsignmentSubmissionType.ts

```diff
@@ -15,20 +15,20 @@ const ArtworkConsignmentSubmissionType = new GraphQLObjectType<
             consignmentSubmission.saleState || consignmentSubmission.state
           const statusDisplayTexts = {
             submitted: "Submission in progress",
-            approved: "Submission in progress",
-            published: "Submission in progress",
-            rejected: "Submission in progress",
+            approved: "Submission evaluated",
+            published: "Submission evaluated",
+            rejected: "Submission evaluated",
             hold: "Submission in progress",
             closed: "Submission evaluated",
             open: "Submission in progress",
             sold: "Sold",
-            "bought in": "Sold",
+            "bought in": "Submission evaluated",
             canceled: "Submission evaluated",
             "withdrawn - pre-launch": "Submission evaluated",
             "withdrawn - post-launch": "Submission evaluated",
           }
```

Comment:
**nitpick:** I would suggest move this to the top of the file instead of the resolved and have the statuses as enums to be imported as well inside the tests to avoid maintaining them up to date